src/Elastic.Apm/Elastic.Apm.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <IsPackable>true</IsPackable>
+    <UseDiagnosticSourceVersionFour>false</UseDiagnosticSourceVersionFour>
```

Comment:
As discussed a small comment would be nice here - or go with a flag which adds the version.

Reply:
Added a comment